src/frontend/components/UI/PlatformFilter/index.tsx

```diff
@@ -14,8 +14,13 @@ export default function PlatformFilter() {
 
   const isMac = platform === 'darwin'
   const isLinux = platform === 'linux'
+  const isWindows = platform === 'win32'
   const disabledIcon = isLinux && category === 'legendary'
 
+  if (isWindows) {
+    return null
```

Comment:
I'm no frontend guy, but wouldn't
```suggestion
    return <></>
```
be better here, since the function is expected to return JSX elements? Or is `null` the same as `<></>`?

Reply:
`null` is better since it won't draw anything. The React fragments return an empty span element.

Reply:
I see
TODO for the future would then be to do this elsewhere as well, I believe we have a few cases in the settings where we draw `<></>` for Linux-specific settings when on Windows